variation-commons-beacon/src/main/java/uk/ac/ebi/eva/commons/beacon/models/GA4GHBeaconQueryResponseV2.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * European Variation Archive (EVA) - Open-access database of all types of genetic
+ * variation data from all species
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.commons.beacon.models;
+
+import uk.ac.ebi.eva.commons.beacon.models.BeaconAlleleRequestBody;
+import uk.ac.ebi.eva.commons.beacon.models.BeaconError;
+import uk.ac.ebi.eva.commons.beacon.models.DatasetAlleleResponse;
+
+import java.util.List;
+
+public class GA4GHBeaconQueryResponseV2 {
```

Comment:
Please change the class name to `BeaconAlleleResponse`. Try using the same names as in the spec

Reply:
fixed